packages/node/src/db/db.module.ts

```diff
@@ -13,6 +12,35 @@ export interface DbOption {
   database: string;
 }
 
+async function timeout(ms) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+async function establishConnection(
+  sequelize: Sequelize,
+  numRetries,
+): Promise<void> {
+  if (numRetries > 0) {
+    try {
+      console.log(
+        'Try to connect to database: ',
+        numRetries,
+        'attempts remaining',
+      );
+      await sequelize.authenticate();
+      console.log('Connection has been established successfully.');
+    } catch (error) {
+      console.error('Unable to connect to the database:', error.message);
```

Comment:
combine these two logs together, e.g: 
```suggestion
      console.error('Unable to connect to the database, will retry: ', error.message);
```

Reply:
Fixed